kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md

```diff
@@ -4,6 +4,9 @@ The sample scripts demonstrate the creation of a WebLogic domain home on an exis
 
 ## Prerequisites
 
+Please read this guide before proceeding.
```

Comment:
Please read this guide before proceeding. -> Before you begin, read this guide, [Domain Resource](../../../../site/domain-resource.md).